CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing to this repository
+
+# Audience
+This set of contributing guidelines is for both external contributors, as well as Verne Global employees. There are number of references to using JIRA issue number in this document. If you are an external contributor and do not have access to the Verne Global JIRA instance, please ignore these references.
+
+# Table of Contents
+1. [Commits and pull requests](#commits-release-notes-pull-requests)
+    1. [Fixing a bug](#fixing-a-bug)
+    2. [Commits](#commits)
+    3. [Pull requests](#pull-requests)
+        1. [Reviewing pull requests](#reviewing-pull-requests)
+        2. [Merging pull requests](#merging-pull-requests)
+
+## Commits and pull requests
+### Fixing a bug
+
+1. Clone the repo, and commit changes to a new branch, named using the convention <JIRA-ISSUE-NUMBER>-description-of-change. e.g. ```HPCDIR-9999-add-contributing```
+2. Make pull requests against the master branch in the repo.
+3. Reference the JIRA issue in both the commit message and the pull request message.
+
+### Commits
+
+Please avoid the following in a commit:
+
+* Mixing whitespace changes with functional code changes.
+* Mixing two unrelated functional changes in the same commit.
+* Sending large new features in a single commit.
+
+Expected git commit message structure (Based on [this article](https://chris.beams.io/posts/git-commit/)):
+
+* The first line should ideally be limited to 50 characters, but must be no more than 70 characters.
+* The first line should not end with a full stop and the first word should be capitalised.
+* Use the imperative mood in the subject line - e.g. ```Fix a typo in CONTRIBUTING.md```, ```Remove if/else block in myfile.sh```, ```Eat your dinner```.
+* Insert a single blank line after the first line.
+* Subsequent lines should be wrapped at 72 characters.
```

Comment:
Just curious why 72? Is that a common standard for git? Not come across it before.

I was going to suggest 79 to stay consistent with Python, but then I noticed 72 is indeed used by Python for "flowing long blocks of text with fewer structural restrictions (docstrings or comments)", so I guess 72 is fine.

Reply:
yeah tbf it's all a little bit archaic and is to do with the old terminal sizes, but it has become a standard of sorts. Personally I don't mind what number it is, as long as we agree and then commit to enforcing the standard.